support/linux/gstreamer/gstreamer.sh

```diff
@@ -0,0 +1,4 @@
+wget https://github.com/ferjm/gstreamer-1.14.1-ubuntu-trusty/raw/master/gstreamer.tar.gz -O gstreamer.tar.gz
```

Comment:
Should I just make this part of the python file?

Also, should we try to host these binaries on our s3 bucket? (I don't know how, but I can ask)